osv/ecosystems/bioconductor.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bioconductor helpers."""
+
+import packaging.version
+import requests
+
+from . import config
+from .helper_base import Ecosystem, EnumerateError
+
+class Bioconductor(Ecosystem):
+  """Bioconductor ecosystem helpers."""
+
+  # Use the Posit Public Package Manager API to pull both the current and archived versions for a specific package
+  # since CRAN doesn't natively support this functionality.
+  _API_PACKAGE_URL_POSIT_BIOCONDUCTOR = 'https://packagemanager.posit.co/__api__/repos/4/packages/{package}?bioc_version={bioc_version}'
+  _BIOC_VERSIONS = ['3.18', '3.17', '3.16', '3.15', '3.14', '3.13', '3.12', '3.11', '3.10', '3.9', '3.8', '3.7', '3.6', '3.5', '3.4', '3.3', '3.2', '3.1']
```

Comment:
Do these `_BIOC_VERSIONS` need to be kept up to date? Is there somewhere we can pull this list from?

Reply:
Yeah, I can pull this dynamically to keep it up to date

Reply:
With 5463e09853c4190de02f1ab00087ffefb3eea7a8, is this still required?